content/en/security/application_security/threats/setup_and_configure.md

```diff
@@ -231,6 +231,46 @@ Support for query strings is not available for Flask.
 
 {{< /programming-lang >}}
 
+{{< programming-lang lang="serverless" >}}
+
+<div class="alert alert-info">ASM support for AWS Lambda is in beta. Threat detection is done by using the Lambda extension.<br><br> Don’t see your desired technology listed here? Datadog is continually adding additional support. Fill out <a href="https://forms.gle/gHrxGQMEnAobukfn7">this short form to send us details</a>.</div>
+
+
+### Supported cloud environments
+
+- AWS Lambda (beta)
+
+### Supported languages and their requirements
+
+Node version x.x
+: If you are bundling using webpack or esbuild, [mark the Datadog libraries as external][4].
+
+Python version x.x
```

Comment:
Remove x.x version number since it's dependent on the extension version, not language